### ISSUE TITLE
Dependencies

### ISSUE THREAD

**Issue by sinys15** [rana/ora#258]
In your description we can see
"The ora package has no external Go dependencies and is available on GitHub and gopkg.in"
but really I see that it get dependencies
>go get gopkg.in/rana/ora.v4
package golang.org/x/sync/errgroup: unrecognized import path **"golang.org/x/sync/
errgroup"** (https fetch: Get https://golang.org/x/sync/errgroup?go-get=1: Proxy A
uthentication Required)
>go get gopkg.in/rana/ora.v4
# cd .; git clone https://go.googlesource.com/sync C:\Users\xxx\go\src\gola
ng.org\x\sync
Cloning into 'C:\Users\xxx\go\src\golang.org\x\sync'...
fatal: unable to access **'https://go.googlesource.com/sync/'**: Received HTTP code
407 from proxy after CONNECT
package golang.org/x/sync/errgroup: exit status 128

Is it possible really to clear code from dependencies?

**Comment by tgulacsi** [comment]
Sure, but cleaning the docs is easier :)

**Comment by sinys15** [comment]
Am I right that this driver want gcc or something like this too?

**Comment by tgulacsi** [comment]
Yes, if you set the CC environment variable to another C compiler
(clang or TDM-GCC64 or MINGW64 for example).

As Oracle does not allow access to its proprietary SQL.NET protocol,
you have  to use OCI (or the Oracle-provided binary-only Java JDBC
driver).
So no, it's not avidable.
sinys15 <notifications@github.com> ezt írta (időpont: 2018. okt. 18.,
Cs, 13:06):
